docs/src/modules/ROOT/pages/upgrading-timefold-solver/upgrade-from-optaplanner.adoc

```diff
@@ -6,10 +6,12 @@
 :icons: font
 
 In spring of 2024, Red Hat announced https://access.redhat.com/articles/7060671[end of life for OptaPlanner].
-Timefold Solver is a faster, feature-rich, and actively developed fork of OptaPlanner.
+Timefold Solver is a faster, feature-rich, and actively developed fork of OptaPlanner by the same team.
 
-Upgrading from OptaPlanner to Timefold Solver Community Edition only takes two minutes.
-Run the command below to upgrade your code automatically.
+== Automatic upgrade
```

Comment:
By splitting up this document in automatic and manual upgrade, it becomes far less daunting to actually execute.